cirq-core/cirq/neutral_atoms/neutral_atom_devices.py

```diff
@@ -26,6 +26,14 @@
     import cirq
 
 
+def _subgate_if_parallel_gate(gate: 'cirq.Gate') -> 'cirq.Gate':
+    """Returns gate.sub_gate if gate is a ParallelGate, else returns gate"""
+    if isinstance(gate, ops.ParallelGate) and not isinstance(gate.sub_gate, MeasurementGate):
```

Comment:
Why the special case here for MeasurementGate?

Reply:
discussed below.

Reply:
As commented below, I'd suggest putting a special case for this inside `validate_operation` and having this function just do `return gate.sub_gate if isinstance(gate, ops.ParallelGate) else gate`.

```python
def validate_operation(self, operation: ops.Operation):
    if not isinstance(operation, (ops.GateOperation, ops.ParallelGateOperation)):
        raise ValueError(f'Unsupported operation: {operation!r}')

    gate = operation.gate
    if isinstance(gate, ops.ParallelGate):
        if isinstance(gate.sub_gate, ops.MeasurementGate):
            raise ValueError(f'ParallelGate over MeasurementGate is not supported: {operation!r}')
        gate = gate.qub_gate

    self.validate_gate(gate)
    ...
```

Reply:
done